packages/zwave-js/src/lib/serialapi/network-mgmt/AddNodeToNetworkRequest.ts

```diff
@@ -142,11 +146,14 @@ export class AddNodeToNetworkRequest extends AddNodeToNetworkRequestBase {
 	public highPower: boolean = false;
 	/** Whether to include network wide */
 	public networkWide: boolean = false;
+	/** Whether to include as long-range or not */
+	public protocolLongRange: boolean = false;
```

Comment:
The ZST39 stick I have refused to honor this bit during normal, insecure inclusion. No clue why. Bug, feature? The spec seems to say it should do it, but I don't know.